OpenID_Connect/oidc_config.sh

```diff
@@ -37,6 +38,123 @@ function print_usage() {
   exit 1
 }
 
+function get_settings_id() {
+  SETTINGS_ID=`curl $CURL_OPTS \
+    -H "Authorization: Bearer $API_TOKEN" \
+    -X GET \
+    ${SETTINGS_ENDPOINT} | jq '.authenticationSettings | .[] | select(.type=="'"${SSO_KEYWORD}"'") | .id'`
+}
+
+function exit_if_no_settings_id() {
+  if [ -z "$SETTINGS_ID" ] ; then
+    echo "No ${SSO_KEYWORD} settings are set"
+    echo
```

Comment:
should we mention how to set the settings?

Reply:
Maybe we can show `Type ./script_name -h for info` message (or similar).
But for me this was a code path taken by the user after deleting the settings, just checking they are, indeed, deleted.

Reply:
Yes. printing the usage string would be enough/good.